pkg/conversion/statev4.go

```diff
@@ -0,0 +1,135 @@
+package conversion
+
+import (
+	"encoding/base64"
+	"encoding/json"
+	"github.com/pkg/errors"
+)
+
+const (
+	AnnotationKeyStateMeta = "tf.crossplane.io/state-meta"
```

Comment:
Is this file excluded from linting? It seems like it would fail due to missing commentary.